### DISCUSSION TITLE
Own Discord? Or other Chat system?

### DISCUSSION THREAD

**Discussion by lampewebdev** [excaliburjs/Excalibur#2150] [Q&A] [answered]
Hey,

I found Excalibur on the weekend after having problems with another HTML engine.
Until now, I'm happy with Excalibur. Thank you for the great work :) 

I would just like to ask:
Are there any plans for a Discord or some other Chat systems where users can join?

**Comment by jedeen** [comment, marked as answer]
Hello, and thanks for the question!

The Excalibur.js team does not have a chat community (Slack, Discord, IRC, etc.).

Running, managing, and moderating a service like that for Excalibur is currently beyond the scope of our available time and resources.
We encourage you to participate in the Discussions space here on GitHub.
